client/src/App.js

```diff
@@ -216,6 +238,7 @@ function App () {
     console.log('EVENT GE_UPDATE_GUESS');
     let message = '';
     if (liveMessage.found) {
+      playAudio('correctGuess');
```

Comment:
@existentialcoder : Is it possible to use the constants defined in AppConstants here instead of hardcoding the names ?

Reply:
Thats what I have used. Can you check.? This is the key for that constant